### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@mmarinchenko please review

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/fundamentals/logging/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/4aebd6b8f9ad34b0bcaeb317320f9987bfa2f6a2/aspnetcore/fundamentals/logging/index.md) | [Logging in .NET Core and ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/logging/index?branch=pr-en-us-30306) |

### CODE REVIEW

**Comment by mmarinchenko** [comment]
@Rick-Anderson 

Now the word "configuring" occurs 2 times🙂 And you forgot about macOS, which is also Unix and uses Bash...

I suggest this:
```
> [!NOTE]
> When configuring environment variables with names that contain `.` (periods) in macOS and Linux, consider the "Exporting a variable with a dot (.) in it" question on **Stack Exchange** and its corresponding [accepted answer](https://unix.stackexchange.com/a/93533).
```

Anyway, thanks! You're the fastest writer in the wild west😉

**Comment by Rick-Anderson** [comment]
@mmarinchenko use the add a suggestion button on the [files tab](https://github.com/dotnet/AspNetCore.Docs/pull/30306/files)

image

**Comment by Rick-Anderson** [comment]
image

File: aspnetcore/fundamentals/logging/index.md [hunk 1]
```diff
@@ -185,7 +185,7 @@ setx Logging__Console__LogLevel__Microsoft.Hosting.Lifetime Trace /M
 ```
 
 > [!NOTE]
-> When configuring environment variables with names that contain `.` (periods), consider the "Exporting a variable with a dot (.) in it" question on **Stack Exchange** and its corresponding [accepted answer](https://unix.stackexchange.com/a/93533).
+> In Linux environments, when configuring configuring environment variables with names that contain `.` (periods), consider the "Exporting a variable with a dot (.) in it" question on **Stack Exchange** and its corresponding [accepted answer](https://unix.stackexchange.com/a/93533).
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/logging/index.md:188]:
```suggestion
> When configuring environment variables with names that contain `.` (periods) in macOS and Linux, consider the "Exporting a variable with a dot (.) in it" question on **Stack Exchange** and its corresponding [accepted answer](https://unix.stackexchange.com/a/93533).
```

Comment (Rick-Anderson) [reply aspnetcore/fundamentals/logging/index.md:188]: With a suggestion I can see the difference and one click accept the suggestion - or all the suggestions I accept.

image

**Pull request merged**